fjord/feedback/views.py

```diff
@@ -45,7 +46,17 @@ def download_firefox(request, template):
 
 
 def thanks(request):
-    template = 'feedback/thanks.html'
+    opinion_id = request.session['opinion_id']
+    user_opinion = Response.objects.get(id=opinion_id)
+    word_count = len(user_opinion.description.split())
+
+    if (waffle.flag_is_active(request, 'thankyou') and
+        user_opinion.locale == 'en-US' and
+        user_opinion.happy == False and
+        word_count >= 7):
+        template = 'feedback/thanks_sad.html'
```

Comment:
The if-stmt isn't readable since it's broken up into lines that are indented at the same place as line 57 which is the if-exp. There are a couple of ways to deal with this. You can either add an extra paren:

```
if ((blah and
     blah and
     blah):
    if-exp here
```

or you can indent an additional 4 spaces:

```
if (blah and
        blah and
        blah):
    if-exp
```

Either one is more readable.